doc/development.md

```diff
@@ -7,7 +7,7 @@ Fork the project on https://github.com/oVirt/vdsm.
 Clone your fork:
```

Comment:
Which commit are You referring to?

Reply:
Every commit should have some text (at least a why) and a signoff (for now). In this case I was referring to the first commit, the other commits are fixes over the first one, so they should probably be squashed.

Personally, I would've amended in this case, so I don't have to squash later. In general, there is a lot of `rebase` involved in the review process :)

Reply:
Ok, it did have a short message and it was simple :)
I've been told not to amend (or force-push ;-) ) in other projects, but I agree it would make sense in this case. Will keep it in mind for next time.

Reply:
In vdsm, engine, ovirt-imageio, and ioprocess we always send good set of changes. We don't add a new commit to fix an issue in a previous commit. We fix issue by "git rebase -i" and editing the commit that need to change.

Reply:
I still don't see the commit message nor the signoff. It is in the PR but is missing from the commit.

Reply:
Is it ok now, @aesteve-rh ? Otherwise I don't understand what I'm missing.

Reply:
That's true. I'll try to include that as well.